DfE.FindInformationAcademiesTrusts/Services/Export/ExportService.cs

```diff
@@ -67,73 +89,245 @@ private static byte[] GenerateSpreadsheet(
         using var workbook = new XLWorkbook();
         var worksheet = workbook.Worksheets.Add("Academies");
 
-        // Adding Trust name and type 
-        worksheet.Cell(1, 1).Value = trustSummary?.Name ?? string.Empty;
-        worksheet.Row(1).Style.Font.Bold = true;
-        worksheet.Cell(2, 1).Value = trustSummary?.Type ?? string.Empty;
+        WriteTrustInformation(worksheet, trustSummary);
+        WriteHeaders(worksheet, headers, 3);
 
-        // Adding Excel headers for the data points for academies
-        for (var i = 0; i < headers.Count; i++)
+        var startRow = 4;
+        for (var i = 0; i < academies.Length; i++)
         {
-            worksheet.Cell(3, i + 1).Value = headers[i];
+            var currentRow = startRow + i;
+            var academy = academies[i];
+            var ofstedData = academiesOfstedRatings.SingleOrDefault(x => x.Urn == academy.Urn);
+            var pupilData = academiesPupilNumbers.SingleOrDefault(x => x.Urn == academy.Urn);
+            var freeMealsData = academiesFreeSchoolMeals.SingleOrDefault(x => x.Urn == academy.Urn);
+
+            GenerateAcademyRow(worksheet, currentRow, academy, ofstedData, pupilData, freeMealsData);
         }
 
-        worksheet.Row(3).Style.Font.Bold = true;
+        worksheet.Columns().AdjustToContents();
+        return SaveWorkbookToByteArray(workbook);
+    }
+
+    private static void GenerateAcademyRow(
+        IXLWorksheet worksheet,
+        int rowNumber,
+        AcademyDetails academy,
+        AcademyOfsted? ofstedData,
+        AcademyPupilNumbers? pupilNumbersData,
+        AcademyFreeSchoolMeals? freeSchoolMealsData)
+    {
+        var previousRating = ofstedData?.PreviousOfstedRating ?? OfstedRating.NotInspected;
+        var currentRating = ofstedData?.CurrentOfstedRating ?? OfstedRating.NotInspected;
+        var percentageFull = CalculatePercentageFull(pupilNumbersData?.NumberOfPupils, pupilNumbersData?.SchoolCapacity);
+
+        SetTextCell(worksheet, rowNumber, 1, academy.EstablishmentName ?? string.Empty);
+        SetTextCell(worksheet, rowNumber, 2, academy.Urn);
+        SetTextCell(worksheet, rowNumber, 3, academy.LocalAuthority ?? string.Empty);
+        SetTextCell(worksheet, rowNumber, 4, academy.TypeOfEstablishment ?? string.Empty);
+        SetTextCell(worksheet, rowNumber, 5, academy.UrbanRural ?? string.Empty);
+
+        SetDateCell(worksheet, rowNumber, 6, ofstedData?.DateAcademyJoinedTrust);
+
+        SetTextCell(worksheet, rowNumber, 7, previousRating.OverallEffectiveness.ToDisplayString() ?? string.Empty);
```

Comment:
previousRating is never null since you default it to `OfstedRating.NotInspected` on line 119 and `ToDisplayString()` always returns a string so the string.empty here is unnecessary.
Same on line 142
```suggestion
        SetTextCell(worksheet, rowNumber, 7, previousRating.OverallEffectiveness.ToDisplayString());
```